nyaa/templates/_formhelpers.html

```diff
@@ -139,3 +139,54 @@
 		{% endif %}
 	</div>
 {% endmacro %}
+
+{% macro render_category_select(name) %}
```

Comment:
I can do that, but `layout.html` adds some conditional logic to determine which category should be selected (i.e. `<option selected />`) based on the search query. This sort of logic isn't really present in the macro, but what I think I can do instead is to enhance it where you can pass an expression that evaluates to a boolean which then determines which option should have the `selected` attribute. Thoughts?